packages/preview-middleware-client/src/messagebundle.properties

```diff
@@ -1,17 +1,17 @@
-QUICK_ACTION_ADD_PAGE_CONTROLLER=Add controller to page
-QUICK_ACTION_SHOW_PAGE_CONTROLLER=Show page controller
+QUICK_ACTION_ADD_PAGE_CONTROLLER=Add Controller to Page
+QUICK_ACTION_SHOW_PAGE_CONTROLLER=Show Page Controller
 QUICK_ACTION_OP_ADD_HEADER_FIELD=Add Header Field
 
-V2_QUICK_ACTION_CHANGE_TABLE_COLUMNS=Change table columns
+V2_QUICK_ACTION_CHANGE_TABLE_COLUMNS=Change Table Columns
```

Comment:
It was earlier feedback from @MN-public to maintain it as sentence case. Did you discuss?

Reply:
Yes, we should use title case here. https://experience.sap.com/fiori-design-web/ui-text-guidelines-for-sap-fiori/#case